configure

```diff
@@ -5701,6 +5705,17 @@ echo "$_discord"
 define_in_config_if_yes $_vkeybd 'ENABLE_VKEYBD'
 define_in_config_if_yes $_eventrec 'ENABLE_EVENTRECORDER'
 
+#
+# Check whether to enable tracy profiler support
+#
+
+if test "$_tracy" = yes ; then
+	append_var DEFINES "-DTRACY_ENABLE"
+fi
+define_in_config_if_yes $_tracy 'ENABLE_TRACY'
+define_in_config_if_yes $_tracy 'TRACY_ON_DEMAND'
+define_in_config_if_yes $_tracy 'FORBIDDEN_SYMBOL_ALLOW_ALL'
```

Comment:
Hack until I know how to solve this in a better way.

Reply:
Do you have any hint for me on how to solve this? I think that profiling should never get delivered in a release build anyway. So maybe activating this isn't a real problem for local builds?

On could just get rid of `common/trace.h` and do the

```c++
#ifdef TRACE_ENABLE
#include "common/tracy/Tracy.hpp"
#endif
```

everywhere in the c++ files where tracing is desired. But that would imo pollute the code with too much Tracy specific stuff. I'm really open for suggestions on how to solve this in a clean and nice way.

Reply:
I just can't find a better way to do this. Is it a problem to activate this for those builds where tracy is enabled? I mean, tracy wouldn't be enabled for the CI builds, so errors would directly pop up (as `FORBIDDEN_SYMBOL_ALLOW_ALL` wouldn't be defined here of course.